pkgs/desktops/gnome/misc/gnome-extensions-cli/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, python3
+, fetchPypi
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

Comment:
Since you used `python3Packages.callPackage` in `all-packages.nix`, you should be able to accept `buildPythonApplication`, `poetry-core`, `colorama`, etc all as arguments at the top of the file

Was there a particular reason to only accept the `python3` argument?

Reply:
There was not, I suppose I did it that way due to inexperience. I can change that.

Reply:
No worries! :)

I was just making sure there wasn't a reason for this before more strongly suggesting a change

Reply:
Let me go ahead and move those to arguments and drop with python3.pkgs.

Reply:
I just pushed. Is this what you were thinking?